src/cdk-experimental/listbox/listbox.spec.ts

```diff
@@ -337,3 +575,48 @@ class ListboxWithOptions {
     this.changedOption = event.option;
   }
 }
+
+@Component({
+  template: `
+    <div cdkListbox
+         [multiple]="isMultiselectable"
+        (selectionChange)="onSelectionChange($event)">
+      <div cdkOption>Purple</div>
+      <div cdkOption>Solar</div>
+      <div cdkOption>Arc</div>
+      <div cdkOption>Stasis</div>
+    </div>`
+})
+class ListboxMultiselect {
+  changedOption: CdkOption;
+  isMultiselectable: boolean = false;
+
+  onSelectionChange(event: ListboxSelectionChangeEvent) {
+    this.changedOption = event.option;
+  }
+}
+
+@Component({
+  template: `
+    <div cdkListbox
+         [useActiveDescendant]="isActiveDescendant"
+    >
```

Comment:
I'm not sure which is the standard, but looks like this closing tag is on the next line, but in the previous template it's on the previous line

Reply:
Yeah I think it should be on the previous line.